pkg/jira/issues.go

```diff
@@ -38,7 +39,14 @@ type jiraAdapter struct {
 }
 
 func (a *jiraAdapter) FindUser(property string) ([]jira.User, *jira.Response, error) {
-	return a.delegate.User.Find(property) // ignore tweaks
+	// the JIRA API does not work as documented, and we can therefore not use the jira.Client.User.Find function here.
+	// This supplies a query parameter to the search endpoint which will result in a 400 as you are required to pass the
```

Comment:
This is slightly confusing to read :P By the `This` you mean `a.delegate.User.Find(`?

Reply:
Yeah, I can clear that up a bit.